src/Domain/Insights/SniffDecorator.php

```diff
@@ -123,11 +122,11 @@ public function getTitle(): string
         $path = explode('\\', $sniffClass);
         $name = (string) array_pop($path);
 
-        $name = Str::before($name, 'Sniff');
+        $name = str_replace('Sniff', '', $name);
```

Comment:
Are you sure we have the same behaviour here?

Reply:
Yes, it's the same behaviour on current version 1.9 : https://github.com/nunomaduro/phpinsights/blob/v1.9.0/src/Domain/Insights/Sniff.php#L65-L67 :wink: